scripts/generate-schemas.py

```diff
@@ -267,14 +284,40 @@ def build_question_properties(question):
     return question_data
 
 
+def build_any_of(any_of, fields):
+    return {
+        'required': [field for field in sorted(fields)],
+        'title': any_of
+    }
+
+
 def build_schema_properties(schema, questions):
+    any_ofs = {}
```

Comment:
Why is `any_ofs` a dictionary? It seems that storing subschemas indexed by question_id with `any_ofs[quesiton.id]` doesn't really add anything, so we might as well start with the list and skip the dictionary sorting and conversion to list